config/prow/plugins.yaml

```diff
@@ -1625,6 +1630,12 @@ external_plugins:
       - issue_comment
       - pull_request
     endpoint: http://cherrypicker  
+  kubernetes-sigs/kueue:
+  - name: cherrypicker
```

Comment:
what does this do? :sweat_smile:

Reply:
see example https://github.com/kubernetes-sigs/node-feature-discovery-operator/pull/190#issuecomment-1476294973

Reply:
is a game changer for branch management

Reply:
after the PR get's merged the bot creates a cherry-pick PR https://github.com/kubernetes-sigs/node-feature-discovery-operator/pull/192

Reply:
:exploding_head: 
great!